include/config/config_graphics.h

```diff
@@ -66,6 +66,10 @@
 // Disable the fix to Koopa's unshelled model.
 #define KOOPA_KEEP_PINK_SHORTS
 
+// Copies the object graph node (model) to the star dance cutscene's object.
```

Comment:
I think this comment should say "Uses the star's model in the start dance cutscene." Things like graph nodes are a bit too in the weeds for end users who just want to change a config setting that has a clear outcome.

Reply:
updated